apps/coscrad-frontend/src/components/AllResources/ResourceInfoPresenter/ResourceInfo.presenter.tsx

```diff
@@ -11,22 +20,26 @@ export const ResourceInfoPresenter = ({
  * TODO [https://www.pivotaltracker.com/story/show/183766033
  * We should expose the Schema as part of our own API docs somehow.
  */
-IAggregateInfo<ResourceType>): JSX.Element => {
-    const resourceIndexLink = (
-        <Link to={`/${routes.resources.ofType(resourceType).index}`}>View {label}s</Link>
-    );
-
-    return (
-        <>
-            <h3>
-                {label} ({resourceType})
-            </h3>
-
-            <div className="resource-meta" data-testid={label}>
-                {resourceIndexLink}
-
-                <div>{description}</div>
-            </div>
-        </>
-    );
-};
+IAggregateInfo<ResourceType>): JSX.Element => (
+    <div>
+        <Card>
+            {/* TODO Handle pluralization properly as soon as we have a Resource Type whose plural form is irregular */}
+            <CardHeader title={`${label}s`} />
+            <CardContent>
+                <div className="resource-meta" data-testid={label}>
+                    {description}
```

Comment:
Another possible way to condense this list to make it more rapid to navigate is to make this description a tooltip.  The names of the Resources are already descriptive so the user will already have an idea of what they're getting into.